softperipheral/doc/sQSPI/nrf54H20_porting_v0_1_0.rst

```diff
@@ -0,0 +1,231 @@
+     - Pull-up
+   * - **P7.3**
+     - IO2
+     - Output
+     - Connected
+     - Pull-up
+   * - **P7.4**
+     - IO3
+     - Output
+     - Connected
+     - Pull-up
+   * - **P7.5**
+     - CSN
+     - Output
+     - Disconnected
+     - No Pull
+
+.. list-table::
+   :widths: 10 10 20 20 20
+   :header-rows: 1
+
+   * - GPIO pin
+     - sQSPI Role
+     - Direction config
+     - Input config
+     - Pin Pull config
+   * - **P9.0**
+     - SCK
+     - Output
+     - Disconnected
+     - No Pull
+   * - **P9.1**
+     - IO0
+     - Output
+     - Connected
+     - Pull-up
+   * - **P9.2**
+     - IO1
+     - Output
+     - Connected
+     - Pull-up
+   * - **P9.3**
+     - IO2
+     - Output
+     - Connected
+     - Pull-up
+   * - **P9.4**
+     - IO3
+     - Output
+     - Connected
+     - Pull-up
+   * - **P9.5**
+     - CSN
+     - Output
+     - Disconnected
+     - No Pull
+
+.. note::
+   sQSPI driver provides a default GPIO configuration/multiplexing for the user, depending on :c:var:`nrfx_qspi2_cfg_t.skip_gpio_cfg` and :c:var:`nrfx_qspi2_cfg_t.skip_pmux_cfg` being set to ``false``.
+
+How to configure pins
+=====================
+
+There are cases in which the sQSPI driver configuration has to be modified, for example: changing pin drive strength to guarantee signal integrity for a new PCB design. You must address these cases on the sQSPI application code:
+
+* If you set the :c:var:`nrfx_qspi2_cfg_t.skip_gpio_cfg` variable to ``true``, the GPIO configuration is not managed by the sQSPI driver and it must be manually handled by the application. This is a requirement for nRF54H20.
+* If you set the :c:var:`nrfx_qspi2_cfg_t.skip_pmux_cfg` variable to ``true``, the GPIO multiplexing is not managed by the sQSPI driver and it must be manually handled by the application. This is a requirement for nRF54H20.
+
+GPIO multiplexing must be handled by setting the correct `CTRLSEL` value in UICR.
+
+The sQSPI application requires FastPad bias configuration to run at high frequency on SCLK.
+
+Memory retention configuration
+******************************
+
+The sQSPI Soft Peripheral requires RAM retention in order to go into the lowest power consumption mode (API call to :c:func:`nrfx_qspi2_deactivate`).
+
+.. _nrf54H20_porting_guide_ram_configuration:
+
+RAM Configuration
+*****************
+
+The sQSPI Soft Peripheral operates directly from non-cached RAM.
+Your build environment must reserve the required RAM and ensure that it is readable/writable by both the Application Core and the FLPR.
+The following is the configuration required for the RAM:
+
+.. list-table:: RAM Configuration Table
+   :widths: auto
+   :header-rows: 1
+
+   * - Start Address
+     - Size
+   * - 0x2f890000
+     - 0x4000
+
+The build environment described in the :ref:`nrf54h20_porting_guide_code` section must comply with this requirements.
+This includes proper settings in linker scripts, device tree specifications (DTS), and resource allocation.
```

Comment:
If it's a general remark, shouldn't this appear somewhere at the beginning of this guide?
```suggestion
The build environment described in the :ref:`nrf54h20_porting_guide_code` section must comply with these requirements.
This includes proper settings in linker scripts, device tree specifications (DTS), and resource allocation.
```